package/MDAnalysis/coordinates/base.py

```diff
@@ -1330,6 +1330,95 @@ def __del__(self):
         self.close()
 
 
+class NewReader(ProtoReader):
```

Comment:
It's just a temporary thing while I try and see what's possible... hopefully it will be the new Reader class across everything